### PR TITLE
new definition for 'react-virtual-keyboardl'

### PR DESCRIPTION
**Pull request by bsurai**
Please fill in this template.

- [ ] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [ ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [ ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [ ] The package does not provide its own types, and you can not add them.
- [ ] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, and `strictNullChecks` set to `true`.

### CODE REVIEW

File: types/react-virtual-keyboard/react-virtual-keyboard-tests.tsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
+import * as React from "react";
+import Keyboard from "react-virtual-keyboard";
+import { ReactKeyboardOptions } from "react-virtual-keyboard";
+
+const kbOptions: ReactKeyboardOptions = {
```
Comment (sheetalkamat) [inline types/react-virtual-keyboard/react-virtual-keyboard-tests.tsx]: Please also add examples from https://www.npmjs.com/package/react-virtual-keyboard

File: types/react-virtual-keyboard/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
+// Type definitions for react-virtual-keyboard 1.0
+// Project: https://www.npmjs.com/package/react-virtual-keyboard
+// Definitions by: Bogdan Surai <https://github.com/bsurai>
```
Comment (sheetalkamat) [inline types/react-virtual-keyboard/index.d.ts]: Please provide better documentation as without the npm package doesnt have source url and we wont be able to verify if you are writing correct definiton

**Review by sheetalkamat — CHANGES_REQUESTED** [review]

File: types/react-virtual-keyboard/index.d.ts (hunk 2 above)
Comment (bsurai) [inline types/react-virtual-keyboard/index.d.ts]: @sheetalkamat Hi. Thanks. I'm not sure what do you mean. There is correct reference to react-virtual-keyboard npm package. What is wrong? Do I need to add READ.me ?

Comment (sheetalkamat) [inline types/react-virtual-keyboard/index.d.ts]: What I meant is that npm package link doesnt have enough information to see if you are declaring correct types. It points to github.com/gaearon/react-hot-boilerplate as source and that seems to be incorrect?

Comment (bsurai) [inline types/react-virtual-keyboard/index.d.ts]: @sheetalkamat I'll write to react-virtual-keyboard authors. They have to change it.

File: types/react-virtual-keyboard/react-virtual-keyboard-tests.tsx (hunk 1 above)
Comment (sheetalkamat) [inline types/react-virtual-keyboard/react-virtual-keyboard-tests.tsx]: @bsurai, we still need this before we can merge/.

Comment (bsurai) [inline types/react-virtual-keyboard/react-virtual-keyboard-tests.tsx]: @sheetalkamat I'll try to finish it this week

Comment (bsurai) [inline types/react-virtual-keyboard/react-virtual-keyboard-tests.tsx]: @sheetalkamat. this one is done

**Pull request merged**